src/EventStore.Core/Services/Transport/Grpc/Streams.Read.cs

```diff
@@ -140,6 +140,8 @@ partial class Streams {
 					_ => throw new InvalidOperationException()
 				};
 
+			context.CancellationToken.Register(() => enumerator.DisposeAsync());
```

Comment:
Does this need to also dispose the registration returned here?

Reply:
yes, fixed in latest commit.